Luna/Luna/OBAddImageViewController.storyboard

```diff
@@ -69,7 +69,7 @@
                                 <color key="textColor" white="0.33333333333333331" alpha="1" colorSpace="calibratedWhite"/>
                                 <nil key="highlightedColor"/>
                             </label>
-                            <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" text="Something about the photo and privacy." textAlignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="71T-tU-uc9">
+                            <textView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" text="This photo is for internal use only." textAlignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="71T-tU-uc9">
```

Comment:
This entire feature will be removed when I merge my branch...but it's alright. It'll be stripped later...